netbox_librenms_plugin/static/netbox_librenms_plugin/js/librenms_sync.js

```
@@ -1442,6 +1636,150 @@ function initializeSyncFormSpinners() {
  * Initialize all sync page functionality.
  * Called on DOMContentLoaded and after HTMX content swaps.
  */
+/**
+ * Wire the "Install Selected" form to collect checked module-table rows before submit.
+ * The form is separate from the table (to avoid nested forms), so we copy the
+ * selected checkbox values into hidden inputs just before the form is submitted.
+ * Guard against duplicate listeners on repeated HTMX swaps via a data attribute.
+ */
+function handleInstallSelectedSubmit() {
```

Biome (2.4.6)

[error] Shouldn't redeclare 'handleInstallSelectedSubmit'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)